lambda_policy.json

```diff
@@ -39,9 +39,9 @@
         {
             "Effect": "Allow",
             "Action": [
-                "ssm:GetParameter"
+                "s3:GetObject"
```

Comment:
Does this update the lambda_execution_role policy ?

Reply:
yeah, it is the policy itself but will remove once we sort our deplyoment out so this repo can be deployment agnostic